src/components/content/geogebra.tsx

```diff
@@ -35,7 +47,9 @@ export function Geogebra({ id }: GeogebraProps) {
       .then((res) => res.json())
       .then((res) => {
         try {
-          const data = res.responses.response.item
+          const _res = res as responseData
+          const data = _res.responses.response.item
```

Comment:
You can add the type to function argument instead:
```tsx
promise.then((res: ResponseData) => {
})    
```

Reply:
nice, tried some things here and forgot to put it back 👍